### PR TITLE
split cmake_check to cmake_check and clang_check

### PR DESCRIPTION
**Pull request by arbipher**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -894,7 +898,7 @@ endif
 CMAKE_DIR_GCC := $(HOME)/pyston-build-gcc
 CMAKE_SETUP_GCC := $(CMAKE_DIR_GCC)/build.ninja
 $(CMAKE_SETUP_GCC):
-	@$(MAKE) cmake_check
+	@$(MAKE) clang_check
```
Comment (kmod) [inline Makefile]: should this be `cmake_check`?

Comment (arbipher) [inline Makefile]: You are right. Sorry for the silly error.

Not quite familiar with github workflow.
Should I add a new commit after your view or change the commit to the correct one? (I choose the latter, but it's troublesome than the former)

**Comment by kmod** [comment]
cool, thanks!

I usually prefer amending the PR rather than adding a new "oops" commit -- `git commit -a --amend` can be a huge time saver with that.

**Comment by arbipher** [comment]
"Oops" Got it.

**Pull request merged**